cypress/integration/page.level.errors.spec.ts

```diff
@@ -98,13 +98,13 @@ describe('Page level error because of ', () => {
 
   it('no namespace in metadata page should show 404', () => {
     // Go to metadata page
-    cy.visit(`/cdap/ns/${FAKE_NAMESPACE}/metadata`);
+    cy.visit(`/metadata/ns/${FAKE_NAMESPACE}`);
```

Comment:
I believe these tests were updated to use the react metadata page where the angular page was down due to issue this PR fixed. Default should be angular until react conversion is complete.